adapters/kubernetes_generic/kubernetes_generic_connect.php

```diff
@@ -125,14 +123,14 @@ public function send($origin, $cmd)
         }
 
         if (($action[1] == "") && ($kube_auth_method != "KUBERNETES" && $kube_auth_method != "EKS")) {
-            $action[2] = $http_protocol . '://' . $this->sd_ip_config . ':5000' . $action[2];
+            $url = $http_protocol . '://' . $this->sd_ip_config . ':5000' . $action[2];
         } else {
-            $action[2] = $kube_http_protocol . '://' . $this->sd_ip_config . ':' . $kube_port . '' . $action[2];
+            $url = $kube_http_protocol . '://' . $this->sd_ip_config . ':' . $kube_port . '' . $action[2];
```

Comment:
What is the use on `. '' .` ?

Reply:
I was focus on changing curl command to curl php function and I didn't check the existing code.
`. '' .` is useless!